packages/salesforcedx-utils-vscode/src/commands/notificationService.ts

```diff
@@ -112,7 +113,7 @@ export class NotificationService {
       executionName
     );
     const showCLISuccessMsg = vscode.workspace
-      .getConfiguration('salesforcedx-vscode-core')
+      .getConfiguration(SFDX_CORE_CONFIGURATION_NAME)
```

Comment:
Since
```
vscode.workspace
      .getConfiguration(SFDX_CORE_CONFIGURATION_NAME)
```

is called twice, how about changing to:

```
const coreConfigurationName = vscode.workspace.getConfiguration(SFDX_CORE_CONFIGURATION_NAME);
const showCLISuccessMsg = coreConfigurationName.get<boolean>('show-cli-success-msg', true);
```

..and then using `coreConfigurationName` on line 132 & 133.